.vimrc

```diff
@@ -0,0 +1,114 @@
+" Use Vim settings, rather then Vi settings. This setting must be as early as
+" possible, as it has side effects.
+set nocompatible
+
+" Leader
+let mapleader = " "
+
+" Switch between the last two files
+nnoremap <leader><leader> <c-^>
+
+" Set undo file to allow undoing even after closing and reopening vim
+silent !mkdir -p ~/.vim/undo > /dev/null 2>&1
+set undofile
+set undodir=~/.vim/undo
+
+" Softtabs, 2 spaces
+set softtabstop=2
+set tabstop=2
+set shiftwidth=2
+set shiftround
+set expandtab
+set backspace=2   " Backspace deletes like most programs in insert mode
+
+" Numbers
+set relativenumber
+set numberwidth=5
+
```

Comment:
It might be useful to include the following:

```
set colorcolumn=80 " highlights the 80th column
set cursorline " highlights the line the cursor is on
set cursorcolumn " highlights the column the cursor is in
```

It seems that setting `cursorline` and `cursorcolumn` could [can cause Vim to respond slowly, especially for large files or files with long lines.](http://vim.wikia.com/wiki/Highlight_current_line), although I don't think we will often, if ever, be reading large files on the server with vim. What do you guys think?